sass/site/primary/_posts-and-pages.scss

```diff
@@ -20,10 +20,54 @@
 	display: inline;
 }
 
+.page,
+.single {
+
+	.entry-header {
+		background: $blue;
+		color: $white;
+		margin-bottom: $size__height--wave;
+		margin-left: calc(50% - 50vw);
+		max-width: none;
+		padding: spacing(5) spacing(2) spacing(4);
+		position: relative;
+		text-align: center;
+		width: 100vw;
+
+		@media (min-width: $size__width--main) {
+			margin-bottom: $size__height--wave-huge;
+		}
+
+		&::after {
+			content: "";
+			background: url("#{$short_wave}") no-repeat top;
+			position: absolute;
+			left: 0;
+			bottom: -$size__height--wave;
+			height: $size__height--wave;
+			width: 100vw;
+
+			@media (min-width: $breakpoint-huge) {
+				background-size: cover;
```

Comment:
You could also do `background-size: 100% 95px;`, which flattens out the curve at really wide screens but at least removes the need to use the `vw` magic 🙂  (just a suggestion, I'm fine either way)

Reply:
I think I like the look of using `vw` since it doesn't flatten it out. What do you think @melchoyce? Here are comparison screenshots.

## vw
![vw](https://user-images.githubusercontent.com/4986487/56708701-499ac580-66e3-11e9-966e-11e4cc7cfb50.png)

## 95px
![95px](https://user-images.githubusercontent.com/4986487/56708707-4e5f7980-66e3-11e9-9abb-4f8a6125b59e.png)

Reply:
I do think I prefer the overall scaling of the `vw` approach. @kjellr, do you have a preference here?

Reply:
I do prefer the height of the `vw` one, but it looks like it flattens out a bit when it starts going down on the right-hand side. Could just be the screenshot. 

Whatever can get us the smoothest curves is all good on my end. 👍